pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "exasol-sphinx-github-pages-generator"
+version = "0.1.0"
+description = "Generates Sphinx GitHub pages for a given Git Repository"
+
+license = "MIT"
+
+authors = [
+    "Marlene Kreß <marlene.kress@exasol.com>"
```

Comment:
I think you can add Torsten as well, he wrote the original code.

Reply:
done